3.4/arm_alpine/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+moodledb_alpine:
+  image: arm32v7/postgres
+  container_name: moodledb_alpine_arm
+  ports:
+    - "4000:5432"
+  environment:
+    - POSTGRES_DATABASE=moodle
+    - POSTGRES_USER=moodle
+    - POSTGRES_PASSWORD=moodle
+moodle_alpine:
+  build: .
+  container_name: moodle_alpine_arm
+  ports:
+    - "8090:80"
+  links:
+    - moodledb_alpine
+  environment:
+    - MOODLE_URL=http://docker.ole.org:80
```

Comment:
We have new convention see https://github.com/ole-vi/moodole/pull/16 and add also config like this https://github.com/ole-vi/moodole/blob/e65dd944d14e26a802364bfab8131a77af8e2623/3.4/x86/config.php